spec/javascripts/tests/MultiTableFilter_spec.js

```diff
@@ -0,0 +1,88 @@
+describe('filter table based on text field criteria', function () {
+  'use strict';
+
+  function visibleRows() {
+    return $('.js-filter-rows').find('tr:visible');
+  };
+
+  function setFilterText(text) {
+    $('.js-filter-input').val(text);
+    $('.js-filter-input').change();
+  };
+
+  describe('on a table', function() {
+    beforeEach(function (done) {
+      var self = this;
+
+      requirejs(['jquery', 'jqueryFastLiveFilter', 'MultiTableFilter'], function ($, jqueryFastLiveFilter, MultiTableFilter) {
```

Comment:
I think we need it so we can set up the component directly on the element, [on this line](https://github.com/moneyadviceservice/rad/pull/221/files#diff-1c467cf6ef9609b95c5021d49af8cd26R20)? We could use the component loader I guess (not sure how) but that seems like it'd introduce something unrelated to the test scenario. Wyt?